operator-framework-spring-boot-starter-test/src/main/java/io/javaoperatorsdk/operator/springboot/starter/test/EnableMockOperator.java

```diff
@@ -0,0 +1,9 @@
+package io.javaoperatorsdk.operator.springboot.starter.test;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import org.springframework.context.annotation.Import;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Import(TestConfiguration.class)
+public @interface EnableMockOperator {}
```

Comment:
You're not injecting operator, it's more like mocking k8s and client, so maybe it's better to call it `InjectMockedKuberentes Client` ?

Reply:
I had named it like this first but then I thought we may want to provide a `MockOperator` as well later on.